### PR TITLE
Logging changed

### PR DESCRIPTION
**Pull request by bulicamihai**

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -224,6 +232,7 @@
   input-imports = [
     "github.com/Financial-Times/base-ft-rw-app-go/baseftrwapp",
     "github.com/Financial-Times/go-fthealth/v1_1",
+    "github.com/Financial-Times/go-logger",
```
Comment (ioanaciont) [inline Gopkg.lock]: Don't forget to delete logrus dependencies

File: content/content_service.go [hunk 2]
```diff
@@ -135,14 +134,14 @@ func (cd service) Write(thing interface{}, transId string) error {
 	labels := `:Content`
 
 	if c.StoryPackage != "" {
-		log.WithField(tid.TransactionIDKey, transId).
+		logger.WithMonitoringEvent("SaveNeo4j", transId, c.Type).
```
Comment (MihaiSorinMoisa) [inline content/content_service.go]: This log should remain the way it was before.

File: content/content_service.go [hunk 3]
```diff
@@ -135,14 +134,14 @@ func (cd service) Write(thing interface{}, transId string) error {
...
 			Infof("There is a story package with uuid=%v attached to Article with uuid=%v", c.StoryPackage, c.UUID)
 		addStoryPackageRelationQuery := addStoryPackageRelationQuery(c.UUID, c.StoryPackage)
 		queries = append(queries, addStoryPackageRelationQuery)
 	}
 
 	if c.ContentPackage != "" {
-		log.WithField(tid.TransactionIDKey, transId).
+		logger.WithMonitoringEvent("SaveNeo4j", transId, c.Type).
```
Comment (MihaiSorinMoisa) [inline content/content_service.go]: This log should remain the way it was before.

**Comment by MihaiSorinMoisa** [comment]
In `content_service.go` on line 164 you should handle the err that `CypherBatch` can throw. If it's an error you should log it using `WithMonitoringEvent`. If the function doesn't return an error you should log the `SaveNeo4j` message.

File: content/content_service.go [hunk 4]
```diff
@@ -95,7 +94,7 @@ func (cd service) Write(thing interface{}, transId string) error {
 
 	// Letting through only articles (which have body), live blogs, content packages, graphics and videos (which don't have a body)
 	if c.Body == "" && !contentTypesWithNoBody[c.Type] {
-		log.WithField(tid.TransactionIDKey, transId).
+		logger.WithMonitoringEvent("SaveNeo4j", transId, c.Type).
```
Comment (MihaiSorinMoisa) [inline content/content_service.go]: I think we should discuss if this log should use `WithMonitoringEvent`. Depending how the monitoring is implemented you might not want to expect a `SaveNeo4j` event for content that isn't supposed to be saved to Neo4j.

File: main.go [hunk 5]
```diff
@@ -90,9 +107,10 @@ func main() {
 			OpenAPIData:   ymlBytes,
 		})
 	}
-	log.SetLevel(log.InfoLevel)
-	log.Infof("Application started with args %s", os.Args)
-	app.Run(os.Args)
+	err :=app.Run(os.Args)
```
Comment (MihaiSorinMoisa) [inline main.go]: Run the command `go fmt` on this file.

**Pull request merged**